src/main/java/ledger/user_interface/ui_controllers/component/IgnoredTransactionTableView.java

```diff
@@ -0,0 +1,110 @@
+    @FXML
+    public TableColumn<IgnoredExpression, String> expressionColumn;
+    @FXML
+    public TableColumn<IgnoredExpression, Boolean> matchOrContainColumn;
+
+
+
+    public IgnoredTransactionTableView() {
+        this.initController(pageLoc, this, "Error creating Payee Editor");
+    }
+
+    @Override
+    public void initialize(URL location, ResourceBundle resources) {
+        this.getSelectionModel().setSelectionMode(SelectionMode.SINGLE);
+        this.setOnKeyPressed(t -> {
+            //Put your awesome application specific logic here
+            if (t.getCode() == KeyCode.DELETE) {
+                handleDelete();
+            }
+        });
+        expressionColumn.setCellValueFactory(new PropertyValueFactory<IgnoredExpression, String>("expression"));
+        expressionColumn.setCellFactory(TextFieldTableCell.forTableColumn());
+        expressionColumn.setOnEditCommit(new EventHandler<TableColumn.CellEditEvent<IgnoredExpression, String>>() {
+            @Override
+            public void handle(TableColumn.CellEditEvent<IgnoredExpression, String> event) {
+                String exp = event.getNewValue();
+                IgnoredExpression igEx = event.getRowValue();
+                igEx.setExpression(exp);
+
+                TaskWithArgs task = DbController.INSTANCE.editIgnoredExpression(igEx);
+                task.startTask();
+                task.waitForComplete();
+            }
+        });
+
+        matchOrContainColumn.setCellValueFactory(new PropertyValueFactory<IgnoredExpression, Boolean>("matchOrContain"));
+        matchOrContainColumn.setCellFactory(ComboBoxTableCell.forTableColumn(new MatchOrContainsStringConverter(), FXCollections.observableArrayList(true,false)));
+        matchOrContainColumn.setOnEditCommit(new EventHandler<TableColumn.CellEditEvent<IgnoredExpression, Boolean>>() {
+            @Override
+            public void handle(TableColumn.CellEditEvent<IgnoredExpression, Boolean> event) {
+                Boolean value = event.getNewValue();
+                IgnoredExpression igEx = event.getRowValue();
+                igEx.setMatchOrContain(value);
+
+                TaskWithArgs task = DbController.INSTANCE.editIgnoredExpression(igEx);
+                task.startTask();
+                task.waitForComplete();
+            }
+        });
+
+        DbController.INSTANCE.registerIgnoredExpressionSuccessEvent(this::asyncUpdateTableView);
+        updateTableView();
+    }
+
+    private void handleDelete() {
+        IgnoredExpression item = (IgnoredExpression)this.getSelectionModel().getSelectedItem();
+        TaskWithArgs<IgnoredExpression> deleteTask = DbController.INSTANCE.deleteIgnoredExpression(item);
+        deleteTask.RegisterFailureEvent((e)->{
+            asyncUpdateTableView();
+            setupErrorPopup("Error deleting transaction.", e);
+        });
+        deleteTask.startTask();
+        deleteTask.waitForComplete();
+
+    }
+
+    public void updateTableView() {
```

Comment:
I renamed the file to get rid of the confusion. this has nothing to do with the TransactionTableView